gp-templates/helper-functions.php

```diff
@@ -341,6 +341,9 @@ function textareas( $entry, $permissions, $index = 0 ) {
 			endforeach;
 
 		endif;
+		if ( ( 'changes_requested' == $entry->translation_status ) && ( ! apply_filters( 'gp_enable_changes_requested_status', false ) ) ) { // todo: delete when we merge the gp-translation-helpers in GlotPress
+			$entry->translations = array();
```

Comment:
Could you add a comment here why this is necessary?

Reply:
Done here https://github.com/GlotPress/GlotPress/pull/1451/commits/d534453390a58ab36e4af669aea9716fe97082ac.